### PR TITLE
Add configuration page docs for Swift

### PR DESCRIPTION
**Pull request by jakecraige**
This can remain open while it's in beta
but I figured I'd open a PR now
since I have the time.

Why:

* Users would like to know how to add Swift to their projects.

This change addresses the need by:

* Adding the Swift section to the configuration page.

screen shot 2015-09-25 at 2 40 18 pm

### CODE REVIEW

**Comment by thorncp** [comment]
LGTM

**Comment by jakecraige** [comment]
Waiting till merge till we complete some internal testing to make sure things work as expected.

**Comment by salbertson** [comment]
@jakecraige what isn't working as expected?

File: app/helpers/configuration_helper.rb [hunk 1]
```diff
@@ -22,4 +22,9 @@ def scss_config_url
   def haml_config_url
     "https://raw.githubusercontent.com/thoughtbot/hound/master/config/style_guides/haml.yml"
   end
+
+  def swift_config_url
+    "https://raw.githubusercontent.com/thoughtbot/hound-swift" \
```
Comment (teoljungberg) [inline app/helpers/configuration_helper.rb]: What do you think about merging this to one line? If that's too long, how about extracting a helper to genearate the url based on the repository and the filepath?

**Comment by salbertson** [comment]
:+1:

**Comment by salbertson** [comment]
@jakecraige is this ready?

**Comment by jakecraige** [comment]
@salbertson We're still testing internally. @gfontenot and @sidraval have been using it on a project this week I believe. Would like to hear their feedback if it's working as expected for them before merging.

**Comment by gfontenot** [comment]
We should add explicit credit to Realm somewhere on here for creating/maintaining SwiftLint

**Comment by jakecraige** [comment]
@gfontenot I think this was discussed briefly with @jpsim before but I don't remember where. We're linking to their repo in 2 places in these docs, do you think we need more?

**Comment by gfontenot** [comment]
When I talked to JP, the request was to have an explicit note about SwiftLint being a Realm project like CocoaDocs is doing with Jazzy (see the bottom of the page on [Argo's docs](http://cocoadocs.org/docsets/Argo/2.2.0/index.html).

**Comment by jpsim** [comment]
> @gfontenot I think this was discussed briefly with @jpsim before but I don't remember where. We're linking to their repo in 2 places in these docs, do you think we need more?

I appreciate linking to the SwiftLint source! I think that's sufficient.

**Comment by jpsim** [comment]
I'd love to see this go live :smile:

**Comment by jakecraige** [comment]
Deployed to prod :) https://houndci.com/configuration#swift

**Pull request merged**